src/Illuminate/Database/Eloquent/Concerns/HasCustomizations.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Concerns;
+
+use Illuminate\Database\Eloquent\Attributes\PrimaryKey;
+use ReflectionClass;
+
+trait HasCustomizations
+{
+    /**
+     * Apply customizations from attributes.
+     *
+     * @return void
+     */
+    protected function applyCustomizations()
```

Comment:
If you rename this method to `initializeHasCustomizations`, it will be called automatically by the `initializeTraits` call on the constructor

Reply:
Oh that's true, forgot about that, thanks!
I'll update now!